doc/cephfs/shell.rst

```diff
@@ -0,0 +1,277 @@
+
+Ceph FS Shell
+=============
+
+The File System (FS) shell includes various shell-like commands that directly interact with the Ceph File System.
+
+Commands
+========
+
+mkdir
+-----
+ Create the directory(ies), if they do not already exist.
+
+ Usage : mkdir [-option] <directory>...
```

Comment:
Put the `Usage` in a box? See: https://github.com/ceph/ceph/commit/8b796173063ac9af8c21364521fc5ee23d901196